drake/solvers/MobyLCP.cpp

```diff
@@ -161,44 +154,43 @@ bool MobyLCPSolver::solve(OptimizationProblem& prog) const {
   // Ms and qs into the appropriate places.  That would be equivalent to this
   // implementation but might perform better if the solver were to parallelize
   // internally.
-  Eigen::VectorXd solution(prog.getNumVars());
+  Eigen::VectorXd solution(prog.num_vars());
   for (const auto& binding : bindings) {
-    Eigen::VectorXd constraint_solution(binding.getNumElements());
+    Eigen::VectorXd constraint_solution(binding.GetNumElements());
     const std::shared_ptr<LinearComplementarityConstraint> constraint =
-        binding.getConstraint();
-    bool solved = lcp_lemke_regularized(
-        constraint->getM(), constraint->getq(),
-        &constraint_solution);
+        binding.constraint();
+    bool solved = SolveLCPLemkeRegularized(
+        constraint->M(), constraint->q(), &constraint_solution);
     if (!solved) {
       return false;
     }
-    binding.writeThrough(constraint_solution, &solution);
+    binding.WriteThrough(constraint_solution, &solution);
   }
-  prog.setDecisionVariableValues(solution);
+  prog.SetDecisionVariableValues(solution);
   return true;
 }
 
 /// Fast pivoting algorithm for denerate, monotone LCPs with few nonzero,
 /// nonbasic variables
-bool MobyLCPSolver::lcp_fast(
-    const Eigen::MatrixXd& M, const Eigen::VectorXd& q, Eigen::VectorXd* z,
-    double zero_tol) const {
+bool MobyLCPSolver::SolveLCPFast(const Eigen::MatrixXd& M,
```

Comment:
Prefer SolveLcpFast to SolveLCPFast.
https://google.github.io/styleguide/cppguide.html#Function_Names